motionplan/tpSpaceRRT.go

```diff
@@ -177,18 +165,29 @@ func (mp *tpSpaceRRTMotionPlanner) plan(ctx context.Context,
 	seedPos := spatialmath.NewZeroPose()
 
 	startNode := &basicNode{q: make([]referenceframe.Input, len(mp.frame.DoF())), cost: 0, pose: seedPos, corner: false}
-	goalNode := &basicNode{q: make([]referenceframe.Input, len(mp.frame.DoF())), cost: 0, pose: goal, corner: false}
+	maps := &rrtMaps{startMap: map[node]node{startNode: nil}}
+	if mp.opt().PositionSeeds > 0 && mp.opt().profile == PositionOnlyMotionProfile {
+		err := maps.fillPosOnlyGoal(goal, mp.opt().PositionSeeds, len(mp.frame.DoF()))
```

Comment:
I think this is going to need to use flipPose somehow inside the implementation of this function

Reply:
It doesn't, because we're in PositionOnlyMode. We don't care about orientation.

Reply:
huh thats interesting